File: FoodGenerator/wwwroot/client-app/src/interfaces/food.ts
export interface Food {
  id: string;
  name: string;
  type: number;
  daysCount: number;
  ingredientsIds: string[];
}

export function isFood(object: any): object is Food {
  return "name" in object && "type" in object && "daysCount" in object;
}

File: FoodGenerator/Database/Models/Food.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace FoodGenerator.Database.Models
{
	public class Food
	{
		[Key]
		[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
		public string Id { get; set; }
		public string Name { get; set; }
		public FoodTypes Type { get; set; }
        public byte DaysCount { get; set; }

        public List<FoodIngredient> FoodIngredients { get; set; } = new List<FoodIngredient>();
        public List<SavedFood> SavedFood { get; set; }

        [NotMapped]
        public List<string> IngredientsIds { get; set; } = new List<string>();
    }
}
